Makefile

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 BUILD_DIR = build
-SERVICES = users things http ws coap lora normalizer influxdb-writer influxdb-reader mongodb-writer mongodb-reader cassandra-writer cassandra-reader postgres-writer postgres-reader cli bootstrap opcua
+SERVICES = users things http ws coap lora influxdb-writer influxdb-reader mongodb-writer mongodb-reader cassandra-writer cassandra-reader postgres-writer postgres-reader cli bootstrap opcua
```

Comment:
If all the addons are added here, probably it would be good to add `transformer` here as well, so that CI can check it and build the docker (for DoeckerHub usage).

Reply:
> If all the addons are added here, probably it would be good to add `transformer` here as well, so that CI can check it and build the docker (for DoeckerHub usage).

Normalizer is removed as a service and we're now using Transformers as packages imported by Message consumers to transform messages. Since there is no standalone Transformer service, we don't have it as an add-on, either.